src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java

```diff
@@ -302,7 +302,7 @@ public void commitWithMessageShouldReturnTheTransaction() throws Exception {
         Transaction transaction = adapter.getResult();
 
         for (Header header : request.getHeaders()) {
-            if (header.getName().compareToIgnoreCase("X-Bitreserve-OTP") == 0) {
+            if (header.getName().compareToIgnoreCase("OTP-Token") == 0) {
```

Comment:
WDYT of changing the condition to `"OTP-Token".equalsIgnoreCase(header.getName())`? It's safer.